### PR TITLE
Fix identifyPorts schema traversal

### PR DESCRIPTION
**Pull request by jpommerening**
The previous code failed to identify `axRole` entries in `patternProperties` and `additionalProperties` because it never descended into a path that is not covered by an entry in the schema's `properties`.

@jonaswjs, @x1B: Can you check if this is correct?

PS: Sorry, but the indent in Line 13ff bothered me and my editors indent detection ;)

### CODE REVIEW

**Comment by x1B** [comment]
Seems reasonable. Not super keen on rebuilding the patternProperties RegExp for each instance key, but the impact is probably negligible compared to e.g. graph layout.

Needs moar changelog, then +1 by me.

**Comment by jpommerening** [comment]
Ok, did that and rebased onto latest master, but unable to test because for unknown reasons the widget does not work atm. Investigating…

File: application/widgets/page-inspector-widget/graph-helpers.jsx [hunk 1]
```diff
@@ -234,12 +235,39 @@ export function graph( pageInfo, options ) {
...
-               const propertySchema = schema.properties[ key ] || schema.additionalProperties;
-               identifyPorts( value[ key ], propertySchema, path.concat( [ key ] ), ports );
+         if( schema.type === 'object' ) {
+            const {
+               properties = {},
+               patternProperties = {},
+               additionalProperties
+            } = schema;
+
+            const propertyPatterns = object.tabulate(
```
Comment (x1B) [inline application/widgets/page-inspector-widget/graph-helpers.jsx:245]: My man!

**Comment by jpommerening** [comment]
After [a](https://github.com/LaxarJS/laxar-developer-tools-content/issues/5) [few](https://github.com/LaxarJS/laxar-developer-tools-content/issues/6) [smaller](https://github.com/LaxarJS/laxar/issues/440) issues identified while (involuntarily) updating to the latest Laxar version, my copy of the laxar-developer-tools-widget is working again.

No changes apart from another rebase onto the latest master where the mentioned issues are fixed.

**Comment by jpommerening** [comment]
Merged on `master`. Thanks to everyone involved! :sparkles:

**Pull request merged**